app/javascript/components/patient/vaccines/VaccineTable.js

```diff
@@ -340,7 +340,7 @@ class VaccineTable extends React.Component {
     return (
       <React.Fragment>
         <Card id="vaccines" className="mx-2 my-4 card-square">
-          <Card.Header className="h5">Vaccinations</Card.Header>
+          <Card.Header className="h5">{this.props.section_label}</Card.Header>
```

Comment:
Add default `|| Vaccinations`.
```suggestion
          <Card.Header className="h5">{this.props.section_label || 'Vaccinations'}</Card.Header>
```

Reply:
@DPC15038

Reply:
done